docs/contributing.rst

```diff
@@ -427,8 +427,12 @@ For squeeze you will need to:
 FreeBSD
 -------
 
-Package installation for FreeBSD uses ``pkg``, not ports.
+Packages can be installed FreeBSD using ``pkg``, 
```

Comment:
I think you're missing a word here - it should be 'can be installed [on|for] FreeBSD'
If you fix that I'd be happy to merge (sorry for the slow review)